content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,204 @@
+|     | **Surface Slope Alignment Min** | Override minimum alignment strength. | Float |
+|     | **Surface Slope Alignment Max** | Override maximum alignment strength. | Float |
+| Slope Filter |     |     |     |
+|     | **Slope Filter Override Enabled** | When enabled, the slope filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Slope Filter Min** | Override minimum slope accepted by filter. | Float |
+|     | **Slope Filter Max** | Override maximum slope accepted by filter. | Float |
+| Surface Mask Filter |     |     |     |
+|     | **Override Mode** | Controls how overrides are considered by the surface mask filter component<br><br>Disable - Overrides are completely ignored<br><br>Replace - Overrides replace those specified in the component<br><br>Extend - Overrides are added to the specified in the component. |     |
+|     | **Inclusion Tags** | A set of surface tags that dictate where vegetation can be placed. | SurfaceTagVector |
+|     | **Exclusion Tags** | A set of surface tags that dictate where vegetation can not be placed. | SurfaceTagVector |
+| Surface Mask Depth Filter |     |     |     |
+|     | **Surface Tags** | A set of surface tags used for depth comparisons. | SurfaceTagVector |
+|     | **Upper Distance Range (m)** | A range used for vertical distance comparisons against the closest point with matching surface tags. | Float |
+|     | **Lower Distance Range (m)** | A range used for vertical distance comparisons against the closest point with matching surface tags. | Float |
+
+File: [`/Gems/Vegetation/Code/Source/Descriptor.cpp`](https://github.com/o3de/o3de/blob/900bd390b9b11c83a77ac3de651dc9eb3d897270/Gems/Vegetation/Code/Include/Vegetation/Descriptor.h)
+
+
+## EBus interface
+
+### Vegetation::AreaInfoBus
+
+| Request Name | Description | Parameters | Return |
+| --- | --- | --- | --- |
+| `GetLayer` |     | None | Float |
+| `GetPriority` | Gets the combined layer and area priority value as a single combined global priority. | None | Float |
+| `GetEncompassingAabb` | Gets the axis-aligned bounding box for the entire vegetation area. | None | AZ::Aabb |
+| `GetProductCount` | Gets the number of currently spawned instances in the vegetation area. | None | AZ::u32 |
+| `GetChangeIndex` | Gets an incrementing number that represents the number of times the Blocker area refreshed since creation. | None | AZ::u32 |
+
+File: [`/Gems/Vegetation/Code/Include/Vegetation/Ebuses/AreaInfoBus.h`](https://github.com/o3de/o3de/blob/0121a0675b53104cfe1d9f6d87bf9b8ac59a4382/Gems/Vegetation/Code/Include/Vegetation/Ebuses/AreaInfoBus.h)
+
+### Vegetation::AreaRequestBus
+
+| Request Name | Description | Parameters | Return |
+| --- | --- | --- | --- |
+| `PrepareToClaim` | Runs any pre-claim checks or logic that is not needed per point. | EntityIdStack& stackIds | Bool |
```

Comment:
"Runs any pre-claim checks or logic that is not needed per point" -> This might make more sense as "Runs any position-independent pre-claim checks or logic"